content/zh/docs/reference/issues-security/security.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: Kubernetes 安全和信息披露
-aliases: [/security/]
```

Comment:
remove ?
it seems that we should sync en/ zh/.
![image](https://user-images.githubusercontent.com/19339970/72427497-d1bb8080-37c6-11ea-961a-c2f0de789ddf.png)

![image](https://user-images.githubusercontent.com/19339970/72427514-d8e28e80-37c6-11ea-84c7-791895e2a4ad.png)

could you sync them? Thanks

Reply:
The aliases are used to create redirects. Ref https://gohugo.io/content-management/urls/#aliases .